corehq/messaging/smsbackends/smsgh/models.py

```diff
@@ -38,7 +40,11 @@ def get_additional_data(self, response):
         except:
             return {}
 
-    def handle_error(self, response):
+    def handle_error(self, response, msg):
+        phone = strip_plus(msg.phone_number)
+        if not (phone.startswith('233') and len(phone) == 12):
```

Comment:
223 means Ghana's phone code, correct? Would a constant be better? (Probably not a big deal assuming smsgh can only be used with Ghana and it was easy enough for me to find)